app/src/main/java/com/gathering/android/event/home/FilterDialogFragment.kt

```diff
@@ -0,0 +1,55 @@
+package com.gathering.android.event.home
+
+import android.app.AlertDialog
+import android.app.Dialog
+import android.os.Bundle
+import androidx.fragment.app.DialogFragment
+import androidx.fragment.app.FragmentManager
+import com.gathering.android.databinding.FrgFilterDialogBinding
+
+
+class FilterDialogFragment(private val eventListViewModel: EventListViewModel) : DialogFragment() {
+
+    private lateinit var binding: FrgFilterDialogBinding
+
+    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
+
+        binding = FrgFilterDialogBinding.inflate(layoutInflater)
+
+        val builder =
+            AlertDialog.Builder(requireContext(), com.gathering.android.R.style.MyAlertDialogStyle)
+        builder.setView(binding.root)
```

Comment:
what happened to setTitle here ?

Reply:
by setting a title (filter by) here , the dialog did not look pretty, instead I created a textview that shows filter by in my XML file.